gnomad_hail/resources/grch38/intervals.py

```diff
@@ -0,0 +1,22 @@
+from gnomad_hail.resources.resource_utils import TableResource
+import hail as hl
+
+lcr = TableResource(
+    path="gs://gnomad-public/resources/intervals/LCRFromHengHg38.ht",
+    import_func=hl.import_locus_intervals,
+    import_args={
+        "path": "gs://gnomad-public/resources/grch38/LCRFromHengHg38.txt",
+        "reference_genome": 'GRCh38',
+        "skip_invalid_intervals": True
+    },
+)
+
+telomeres_and_centromeres = TableResource(
+    path='gs://gnomad-public/resources/intervals/hg38.telomeresAndMergedCentromeres.ht',
+    import_func=hl.import_bed,
+    import_args={
+        "path": 'gs://gnomad-public/resources/grch38/hg38.telomeresAndMergedCentromeres.bed',
+        "reference_genome": 'GRCh38',
+        "skip_invalid_intervals": True
+    }
+)
```

Comment:
I think we should either move these into `reference_data.py` or move all the interval files out and into this module like `syndip_hc_intervals` I'm fine either way but would prefer to be consistent across the modules as to what goes where. If we chose two separate files, I think 37 should be adjusted as well to have a separate `intervals`module.